packages/expo-gl-cpp/cpp/EXGLContext.h

```diff
@@ -23,9 +23,9 @@
 
 #include <jsi/jsi.h>
 
-#include "EXGLNativeMethodsUtils.h"
-#include "EXJSIUtils.h"
+#include "EXJsiUtils.h"
```

Comment:
I have no conviction to `JSI` -> `Jsi` 😅 If you really want to change it, maybe `EXJavaScriptInterfaceUtils`?